### DISCUSSION TITLE
Spatial 3D audio issue when in third person

### DISCUSSION THREAD

**Discussion by richie64** [spatialsys/spatial-unity-sdk#17] [Help] [answered]
I am prototyping a musical experience with speakers positioned in different parts of the scene, each playing a different kind of music, the idea is that when the avatar moves near the speaker the volume will go up and you can hear the music.

The problem is that when in third person spatial positions Unity's Audio Listener in the camera and not in the avatar itself, that means that even if you move your avatar right next to the speaker you might not hear the music depending on the position of the camara, or you may also position the avatar far away from the speaker but still hear the music when moving the camera.

Is there a way to repossition the Audio Listener on the avatar? If not, how can I submit a bug report for this issue?

Here is a video showing the problem:

[https://youtu.be/hhSm9upomTs](https://youtu.be/hhSm9upomTs)

**Comment by AlanSpatial** [comment, marked as answer]
For a temporary solution you can try limiting the max zoom distance of the camera, or even set it to 0 to force first person mode.

https://cs.spatial.io/reference/SpatialSys.UnitySDK.ICameraService.maxZoomDistance

No need to submit an issue, I've noted it.

**Reply by richie64** [reply]
Hey, thank you so much for the quick response, I am building this for a client and forcing first person mode is honestly not ideal, is there a time frame for when we might see the fix?
If it's going to take too much I might have to develop a custom solution using 2D audio and adjusting the volume of each speaker depending on the distance to the avatar, that should work, but using Unity's own solution would be so much better.

 Thank you again for the help!

**Reply by AlanSpatial** [reply]
I wouldnt expect a fix for this in the immediate future.

**Reply by richie64** [reply]
oh, that's a shame, thank you for the clarification though so I can start working on a custom solution
